### PR TITLE
Add Open Carousel

### PR DESCRIPTION
**Pull request by marcod1419**
This PR adds the "Open" variant for Carousel. E2E tests and unit tests will be expanded upon once the "Card" variant is in development. Additionally, docs are still awaiting assets, so the sandbox content is not final.

### CODE REVIEW

File: packages/ContentCarousel/ContentCarousel.jsx [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+          handleSlideTransition('left', 1)
+        } else if (movement[0] > 0 && currentPage > 1) {
+          handleSlideTransition('right', -1)
+        }
+      }
+    },
+    { filterTaps: true, axis: 'x', threshold: 100 }
+  )
+
+  const preloadImage = url => {
```
Comment (jraff) [inline packages/ContentCarousel/ContentCarousel.jsx:184]: Out of curiosity, what do the `preloadImage()`, `findImages()`, and the `useEffect()` do?

Comment (marcod1419) [reply packages/ContentCarousel/ContentCarousel.jsx:184]: These functions load the `picture` prop of every slide on initial render. It stops images from loading in only on slide switch.

File: packages/ContentCarousel/PageIndicator/PageIndicator.jsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  })
+  return (
+    <Dot onClick={onClick} isCurrentPage={isCurrentPage} aria-current={isCurrentPage}>
+      <FocusOutline />
+    </Dot>
+  )
+}
+
+const PageIndicator = ({ currentPage, totalPages, changePage, handleSlideSkip, ...rest }) => {
+  const generateDots = () => {
```
[the remark this answers is not in the document]
Comment (marcod1419) [reply packages/ContentCarousel/PageIndicator/PageIndicator.jsx:70]: I think memoization is unnecessary. The page indicator will frequently change props as the user navigates the component, making the gains from using `React.memo` pretty marginal IMO. `generateDots` is in a function just so we don't have a lot of logic in the component's JSX.

**Comment by theetrain** [comment]
A new PR will be made.

**Pull request closed without merging**